dChatServer/ChatPacketHandler.cpp

```diff
@@ -113,6 +114,7 @@ void ChatPacketHandler::HandleFriendRequest(Packet* packet) {
 	inStream.Read(isBestFriendRequest);
 
 	auto requestor = playerContainer.GetPlayerData(requestorPlayerID);
+	if (requestor->playerName == playerName) return;
```

Comment:
Not blocking, but would it make sense to have this return a general error to the client, for the sake of it?

Reply:
good idea.  Should implement that as well.